packages/lib/timeFormat.ts

```diff
@@ -6,23 +6,35 @@
  */
 import { localStorage } from "@calcom/lib/webstorage";
 
+const is24hLocalstorageKey = "timeOption.is24hClock";
+
+export const setIs24hClockInLocalStorage = (is24h: boolean) =>
+  localStorage.setItem(is24hLocalstorageKey, is24h.toString());
+
+export const getIs24hClockFromLocalStorage = () => {
+  const is24hFromLocalstorage = localStorage.getItem(is24hLocalstorageKey);
+  if (is24hFromLocalstorage === null) return null;
+
+  return is24hFromLocalstorage === "false" ? false : true;
```

Comment:
```suggestion
  return is24hFromLocalstorage === "true";
```

Reply:
With only comparing to true should return true|false as null it's being handled before.

Reply:
Yes smart!!